Makefile

```diff
@@ -105,3 +105,12 @@ deploy: examples/admission-cert.pem
 		|sed "s/__admission_cert__/$(shell cat examples/admission-cert.pem|base64 -w0)/g" \
 		|sed "s/__admission_key__/$(shell cat examples/admission-key.pem|base64 -w0)/g" \
 		|kubectl apply -f -
+
+lint:
+	./hack/verify-type-revision-annotation-const.sh
+	gometalinter --config gometalinter.json ./...
+
+check-dependencies:
+	apt update && apt install -y mercurial
```

Comment:
Can we add a check that first checks if the  binaries exist before trying to install them?